_posts/2019-07-11-deploy-spring-boot-kubernetes.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+authors: [yolan_vloeberghs]
+title: 'Deploying your Spring Boot application in the cloud with Kubernetes'
+image: /img/2019-07-11-deploy-spring-boot-kubernetes/banner.jpg
+tags: [Kubernetes,Spring,Spring Boot,Docker,AWS,EKS,Google,GKE,DigitalOcean,Azure,AKS,kubectl,Minikube]
+category: Cloud
+comments: true
+---
+
+# Deploying your Spring Boot application in the cloud with Kubernetes
+
+Assuming you have already heard of Kubernetes, you are probably aware of the continuing growth of this platform. More and more platforms are growing in popularity because of the flexibility of these technologies. Look at OpenShift, Cloud Foundry, ....
+
+As adaptation is growing, many developers are wondering how they effectively use this platform to deploy their application in the cloud with a Kubernetes cluster and use its benefits.
+
+Many big providers have already picked up Kubernetes and are providing their own implementations. A couple of examples are [Amazon Web Services (EKS)](https://aws.amazon.com/eks/){:target="_blank" rel="noopener noreferrer"}, [Google Cloud Platform (GKE)](https://cloud.google.com/kubernetes-engine/){:target="_blank" rel="noopener noreferrer"}, [Azure (AKS](https://azure.microsoft.com/nl-nl/services/kubernetes-service/){:target="_blank" rel="noopener noreferrer"}, [DigitalOcean](https://www.digitalocean.com/products/kubernetes/){:target="_blank" rel="noopener noreferrer"}, .... 
+
+## Prerequisites
+* [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/){:target="_blank" rel="noopener noreferrer"}
+* A Spring Boot project
+* A Kubernetes cluster in the cloud (you can use [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/){:target="_blank" rel="noopener noreferrer"} if you want to try this on your local machine)
```

Comment:
```suggestion
* A Kubernetes cluster. This can be a cluster in the cloud, in an on premise datacenter or you can use [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/{:target="_blank" rel="noopener noreferrer"} if you want to try this on your local machine)
```